service_container/3.3-di-changes.rst

```diff
@@ -0,0 +1,740 @@
+The Symfony 3.3 DI Container Changes Explained (autowiring, _defaults, etc)
+===========================================================================
+
+If you look at the ``services.yml`` file for a new Symfony 3.3 project, you'll
+notice some big changes: ``_defaults``, ``autowiring``, ``autoconfigure`` and more.
+These features are designed to *automate* configuration and make development faster,
+without sacrificing predictability, which is very important! Another goal is to make
+controllers and services behave more consistently. In Symfony 3.3, controllers *are*
+services by default.
+
+The documentation has already been updated to assume you have these new features enabled.
+If you're an existing Symfony user and want to understand the "what" and "why" behind
+these changes, this chapter is for you!
+
+All Changes are Opt-In
+----------------------
+
+Most importantly, **you can upgrade to Symfony 3.3 today without making any changes to your app**.
+Symfony has a strict :doc:`backwards compatibility promise </contributing/code/bc>`,
+which means it's always safe to upgrade across minor versions.
+
+All of the new features are **opt-in**: you need to actually change your configuration
+files to use them.
+
+The new default services.yml File
+---------------------------------
+
+To understand the changes, look at the new default ``services.yml`` file in the
+Symfony Standard Edition:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/services.yml
+        services:
+            # default configuration for services in *this* file
+            _defaults:
+                # automatically injects dependencies in your services
+                autowire: true
+                # automatically registers your services as commands, event subscribers, etc.
+                autoconfigure: true
+                # this means you cannot fetch services directly from the container via $container->get()
+                # if you need to do this, you can override this setting on individual services
+                public: false
+
+            # makes classes in src/AppBundle available to be used as services
+            # this creates a service per class whose id is the fully-qualified class name
+            AppBundle\:
+                resource: '../../src/AppBundle/*'
+                # you can exclude directories or files
+                # but if a service is unused, it's removed anyway
+                exclude: '../../src/AppBundle/{Entity,Repository}'
+
+            # controllers are imported separately to make sure they're public
+            # and have a tag that allows actions to type-hint services
+            AppBundle\Controller\:
+                resource: '../../src/AppBundle/Controller'
+                public: true
+                tags: ['controller.service_arguments']
```

Comment:
it has been tagged *only* if it extends the base Controller or AbstractController
but since this is not a hard requirement, this tag is still required (thus your PR is bogus)